dell/xps/15-7590/default.nix

```diff
@@ -21,6 +21,9 @@
   # Prevent small EFI partiion from filling up
   boot.loader.grub.configurationLimit = 10;
 
+  # This will save you money and possibly your life!
+  services.thermald.enable = lib.mkDefault true;
```

Comment:
`lib` needs to be added to the header of this module:

```
+ { lib, ...}:
```

Reply:
my bad, missed it when copy pasting from my config